File: helpers/name_generator.py
from random import random

namesn = ["Свято", "Яро", "Влади", "Изя", "Вяче", "Мечи", "Бори", "Гори", "Бого", "Любо","Мило","Свето","Миро","Добро","Брони"]
namesk = ["слав", "мир", "полк", "люб", "рад", "мил", "зар", "мысл", "дан", "гор", "яр", "вед", "бор", "мысл", "свет"]

def get_fullname(gender):
    return get_name(gender)+' '+get_surname(gender)

def get_surname(gender):
    n = round(random()*15)
    k = round(random()*15)
    s1 = namesn[n-1]+namesk[k-1]
    if var2 == "м":
        s = s1 + "ич"
    elif var2 == "ж":
        s = s1 + "ична"
    else:
        s = "NaN"
    return s


def get_name(gender):
    n = round(random()*15)
    k = round(random()*15)
    s1 = namesn[n-1]+namesk[k-1]
    if var2 == "м":
        s = s1
    elif var2 == "ж":
        s = s1+"а"
    else:
        s = "NaN"
    return s

File: BrocaZone/helpers/NameGenerator.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BrocaZone.helpers;

public static class NameGenerator{
    static readonly string[] namesN = {"Свято", "Яро", "Влади", "Изя", "Вяче", "Мечи", "Бори", "Гори", "Бого", "Любо", "Мило", "Свето", "Миро", "Добро", "Брони"};
    static readonly string[] namesK = {"слав", "мир", "полк", "люб", "рад", "мил", "зар", "мысл", "дан", "гор", "яр", "вед", "бор", "мысл", "свет"};

    public static string GetFullName(bool isMale)
        {
            return GetName(isMale) + " " + GetSurname(isMale);
        }

    static string GetName(bool isMale)
        {
            Random rnd = new Random();
            int n = rnd.Next(1, 16);
            int k = rnd.Next(1, 16);
            string surname = namesN[n - 1] + namesK[k - 1];
            if (isMale)
            {
                return surname;
            }
            else
            {
                return surname + "а";
            }
        }

    static string GetSurname(bool isMale)
        {
            Random rnd = new Random();
            int n = rnd.Next(1, 16);
            int k = rnd.Next(1, 16);
            string surname1 = namesN[n - 1] + namesK[k - 1];
            string suffix = "";
            if (isMale)
            {
                suffix = "ич";
            }
            else
            {
                suffix = "ична";
            }

            return surname1 + suffix;
        }
    }



